lld/docs/ELF/linker_script.rst

```diff
@@ -198,13 +198,52 @@ the current location to a max-page-size boundary, ensuring that the next
 LLD will insert ``.relro_padding`` immediately before the symbol assignment
 using ``DATA_SEGMENT_RELRO_END``.
 
+Section Classes
+~~~~~~~~~~~~~~~
+
+``SECTIONS`` commands can define classes of input sections:
```

Comment:
`The CLASS keyword inside a SECTIONS command can define`

Reply:
Done; also removed unnecessary `can` tense.